omnigibson/utils/object_state_utils.py

```diff
@@ -97,17 +139,6 @@ def sample_kinematics(
         og.sim.step_physics()
 
         # This would slightly change because of the step_physics call.
-        old_pos, orientation = objA.get_position_orientation()
```

Comment:
again, this line shouldn't be removed.

Reply:
Sorry - I've fixed this before but something very odd is happening with this file (I'm actually not seeing the diff here locally - vscode is thinking a / version and a \ version of the path are different). I've manually saved it again - resolve if it's good now